client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -372,6 +373,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		/>
 	);
 
+	const blankCanvasContent = isEnabled( 'signup/design-picker-build-your-own' ) ? (
+		<div className="blank-canvas-cta-wrapper">
+			<div className="blank-canvas-cta__image-wrapper">
+				<img
+					className="blank-canvas-cta__image"
+					src={ blankCanvasImageUrl }
+					alt="Blank Canvas Header"
+				/>
+			</div>
+			<h3 className="blank-canvas-cta__title"> { translate( 'Start with a blank canvas' ) } </h3>
+			<p className="blank-canvas-cta__subtitle">
+				{ translate(
+					"Can't find something you like? Create something of your own by mixing and matching patterns"
+				) }
+			</p>
+			<Button
+				className="blank-canvas-cta__button"
+				onClick={ () => goToStep?.( 'buildYourOwnLayout' ) }
```

Comment:
We are using `buildYourOwnLayout` for the name of the step, and `signup/design-picker-build-your-own` for the flag name.

For more clarity, we could use the className `build-your-own-cta` instead of `blank-canvas-cta` because **blank-canvas** is not connected to the flag or the step name.